main.js

```diff
@@ -122,20 +121,31 @@ function hideImage() {
   bellImage.classList.add('hidden');
   viewMessage.classList.remove('hidden');
 }
+
 function hideForm() {
   createMessageForm.classList.add('hidden');
 }
 
 function hideErrorMessage() {
-  console.log('hide error message function')
   textInputLabel.classList.add('hidden');
   errorMessage.classList.add('hidden');
 }
 
 function showBellImage() {
-  console.log("clear button function")
   hideForm();
   hideErrorMessage();
   bellImage.classList.remove('hidden');
   viewMessage.classList.add('hidden');
 }
+
+function updateMantras() {
```

Comment:
I like that you added this bit to push mantras into the data model, however you're not technically *rendering* the message from the data model at the moment. Just something to think about.